common/components/controllers/AboutProjectController.jsx

```diff
@@ -228,29 +228,21 @@ class AboutProjectController extends React.PureComponent<{||}, State> {
     );
   }
 
-  _renderPositionsOld(): ?Array<React$Node> {
-    const project = this.state.project;
-    return project && project.project_positions && project.project_positions.map((position, i) =>
-      <div key={i}>
-        <span className="pseudo-link" onClick={this.showPositionModal.bind(this,position)}>{position.roleTag.display_name}</span>
-      </div>
-    );
-  }
-
   _renderPositions(): ?Array<React$Node> {
     const project = this.state.project;
-    return project && project.project_positions && project.project_positions.map((position, i) => {
-      const positionDisplay = position.roleTag.subcategory + ":" + position.roleTag.display_name;
-      return (
-          <div key={i}>
-          {
-            position.descriptionUrl
-            ? <a href={position.descriptionUrl} target="_blank" rel="noopener noreferrer">{positionDisplay}</a>
-            : <span>{positionDisplay}</span>
-          }
-          </div>
-        );
-    });
+    return project && project.project_positions && _.chain(project.project_positions).sortBy('roleTag.tag_name').value()
```

Comment:
tag_name is a unique identifier for a tag that doesn't generally factor in the subcategory.  We'll want to sort by the subcategory + display name.